docs/user-guide/creating-application/workflow/ci-pipeline.md

```diff
@@ -164,16 +137,65 @@ Select the appropriate filter and pass the matching condition as a regular expre
 
 Select **Create Pipeline**.
 
-> **Note**
->
-> **(a)** You can provide pre-build and post-build stages via the Devtron tool’s console or can also provide these details by creating a file `devtron-ci.yaml`
-> inside your repository. There is a pre-defined format to write this file. And we will run these stages using this YAML file.
-> You can also provide some stages on the Devtron tool’s console and some stages in the devtron-ci.yaml file. But stages defined through the `Devtron` dashboard are
-> first executed then the stages defined in the `devtron-ci.yaml` file.
->
-> **(b)** The total timeout for the execution of the CI pipeline is by default set as 3600 seconds. This default timeout is configurable according to the use case. The timeout can be edited in the configmap of the orchestrator service in the env variable as `env:"DEFAULT_TIMEOUT" envDefault:"3600"`
+{% hint style="info" %}
+**(a)** You can provide pre-build and post-build stages via the Devtron tool’s console or can also provide these details by creating a file `devtron-ci.yaml` inside your repository. There is a pre-defined format to write this file. And we will run these stages using this YAML file. You can also provide some stages on the Devtron tool’s console and some stages in the devtron-ci.yaml file. But stages defined through the `Devtron` dashboard are first executed then the stages defined in the `devtron-ci.yaml` file.
```

Comment:
There is a mention about `devtron-ci.yaml` file which has a predefined format, but no mention of where can I find it or how can I use it.